lib/core/src/model.rs

```diff
@@ -1288,7 +1303,10 @@ pub enum PaymentDetails {
         /// Represents the invoice description
         description: String,
 
-        /// In case of a Send swap, this is the preimage of the paid invoice (proof of payment).
+        /// The block at which the swap will no longer be valid
+        expiration_block: u32,
```

Comment:
NIT: Maybe `expiration_height` or `expiration_blockheight`? I know the type makes it evident this isn't a block hash, but specifying height would be better IMO.